docs/app-integration-development/12-App-OAuth-Token.md

```diff
@@ -11,23 +11,30 @@ tags: [app-integration-development]
 > require a feature flag before this functionality is available to you. Please reach out to us if you have any questions or
 > need support.
 
-## Register an App
-Scoped OAuth clients allow your application to act on a PagerDuty Account as a PagerDuty App. Your application's access to the PagerDuty Account is controlled by the scopes it is granted. Before you start building, you first need to [register a PagerDuty App](03-Register-an-App.md), then [add Scoped OAuth functionality](04-App-Functionality.md). This is done via the Developer Mode UI in your PagerDuty Account.
+<!-- theme:warning -->
+> ### This flow is only for server-side apps with Scoped OAuth functionality
+> A client_secret should be treated as a password and stored securely. It should never be stored in public code.
 
-The `client_id`, `client_secret`, and all selected scopes will be used to obtain an access token.
+## About App OAuth Tokens
+
+To act on a PagerDuty Account as a PagerDuty App, you must obtain an app token for that account. A token is only issued if the application has been previously granted access to that account. Applications are implictly granted access to the account that created it.
```

Comment:
```suggestion
To act on a PagerDuty Account as a PagerDuty App, you must obtain an app token for that account. A token can only be issued against the account that created it.
```

Since there's no publish process just yet.